.circleci/real_config.yml

```diff
@@ -3171,11 +3171,11 @@ jobs:
             - run:
                 name: Start defaultrm minikube
                 command: |
-                  source tools/k8s/launch-minikube-with-gateway.sh defaultrm
+                  K8S_VERSION=1.29.5 source tools/k8s/launch-minikube-with-gateway.sh defaultrm
```

Comment:
totally agree. 

We need some way for updating it as new versions come out since previously it would just fall out of date then the tests would fail so we unpinned the version.